src/status_im2/contexts/quo_preview/preview.cljs

```diff
@@ -221,39 +220,39 @@
                :blur-amount   10
                :overlay-color (colors/theme-colors colors/white-opa-70 colors/neutral-80-opa-80)}
```

Comment:
nit, could you pass `theme` here?

Reply:
Not quite, because we eventually want to make that argument mandatory for that function and remove that internal ratom

Reply:
The users theme should still come from the theme provider if/when everything is eventually set right.

Reply:
@J-Son89 I'll remember to take this in consideration when doing other changes in previews, because this wouldn't be the only place to change, so I would like to apply it cleanly and with the theme provider surrounding preview screens.

Side note: I find confusing to talk in PRs about the future signature of a function. I believe it would be clearer for everybody if we just opened a quick PR to add a new `theme-colors` function and deprecated the old one with `:deprecated` metadata. Once no more warnings appear in `make lint` about the usage of the old function, then we just remove it. This strategy works well for basically any function. This way, we can progressively change the code and hopefully it helps when we all communicate in PRs.

Reply:
Yep, for sure. Sounds good to me 👍